messaging/amqp-reactive/src/test/java/io/quarkus/ts/messaging/amqpreactive/ProdAmqpReactiveIT.java

```diff
@@ -42,7 +43,7 @@ public void testContextPropagation() {
         String operationName = "GET /price";
         String lookback = "1h";
         String serviceName = "messaging-amqp-reactive";
-        await().atMost(5, TimeUnit.SECONDS).pollInterval(Duration.ofSeconds(1)).untilAsserted(() -> {
+        await().atMost(10, TimeUnit.SECONDS).pollInterval(Duration.ofSeconds(1)).untilAsserted(() -> {
```

Comment:
It's better than a flaky test, but what exactly could take 10 seconds? I think the cause of this change is incorrect delay of the traces exporter